generics/services/user-management.js

```diff
@@ -39,191 +39,215 @@ var platformUserProfile = function ( userId,token ) {
             }
 
             request.get(
-                platformUserRolesUrl,{
-                    headers: {
-                        "internal-access-token": process.env.INTERNAL_ACCESS_TOKEN,
-                        "X-authenticated-user-token" : token 
-                    }
-                },
+                platformUserRolesUrl, {
+                headers: {
+                    "internal-access-token": process.env.INTERNAL_ACCESS_TOKEN,
+                    "X-authenticated-user-token": token
+                }
+            },
                 _userManagementCallBack
             )
-
         } catch (error) {
             return reject(error);
         }
-    })
-
+    });
 }
 
 /**
-  * post create PlatForm User 
+  * To create platForm User 
   * @function
-  * @name createPlatFormUser
-  * @returns {Promise} returns a promise.
+  * @name createPlatformUser
+  * @param userDetails - user details 
+  * @param token - logged in user token  
+  * @returns {json} returns created user details
 */
 
-var createPlatFormUser = function ( requestBody,token ) {
-
-    const platformUserRolesUrl = 
-    urlPrefix + constants.endpoints.PLATFORM_USER_CREATE;
-    
+const createPlatformUser = function (userDetails, token) {
     return new Promise(async (resolve, reject) => {
         try {
 
-
+            let platformUserRolesUrl =
+                apiBaseUrl + constants.endpoints.PLATFORM_USER_CREATE;
             let options = {
-                "headers":{
-                "content-type": "application/json",
-                "authorization" :  process.env.AUTHORIZATION,
-                "x-authenticated-user-token" : token,
+                "headers": {
+                    "content-type": "application/json",
+                    "authorization": process.env.AUTHORIZATION,
+                    "x-authenticated-user-token": token,
                 },
-                json : requestBody
+                json: userDetails
```

Comment:
@rakeshSgr - Can we have one API call function so that setting options is not repeated for every function in this file

Reply:
@rakeshSgr - Similar to done in samiksha service file in this same PR